src/pages/student/main/Registration/index.js

```diff
@@ -61,150 +63,135 @@ const Registration = () => {
-                    <Form.Control
-                      onChange={onChange}
-                      value={value}
-                      ref={ref}
-                      className="cntrs"
-                      type="text"
-                      placeholder="e.g. jhondoe"
-                      isInvalid={!!errors?.name}
-                      required
-                      maxLength={50}
-                    />
-                  )}
-                />
-                <Form.Control.Feedback type="invalid">
-                  {errors?.name}
-                </Form.Control.Feedback>
-              </Form.Group>
+              <h4>Sign Up</h4>
+            </div>
+            <Form
+              onSubmit={handleSubmit(handleOnSubmit)}
+              className={style.form}
+            >
+              <section>
+                <Form.Group className="mb-4" controlId="Name">
+                  <Form.Label>
+                    <h6 className="mb-0">Name</h6>
+                  </Form.Label>
+                  <Controller
+                    control={control}
+                    name="name"
+                    render={({ field: { onChange, value, ref } }) => (
+                      <InputField
+                        ref={ref}
+                        type="text"
+                        value={value}
+                        onChange={onChange}
+                        fieldSize="md"
+                        isInvalid={!!errors?.name}
+                        placeholder="e.g. John Doe"
+                        maxLength={50}
+                      />
+                    )}
+                  />
+                  <Form.Control.Feedback type="invalid">
+                    {errors?.name}
+                  </Form.Control.Feedback>
+                </Form.Group>
 
-              <Form.Group className="mb-4" controlId="Email">
-                <Form.Label>
-                  <h6 style={{ marginBottom: '0px' }}>Email</h6>
-                </Form.Label>
-                <Controller
-                  control={control}
-                  name="email"
-                  defaultValue=""
-                  render={({ field: { onChange, value, ref } }) => (
-                    <Form.Control
-                      onChange={onChange}
-                      value={value}
-                      ref={ref}
-                      className="cntrs"
-                      type="email"
-                      placeholder="e.g. jhondoe@gmail.com"
-                      isInvalid={!!errors?.email}
-                      required
-                      maxLength={50}
-                    />
-                  )}
-                />
-                <Form.Control.Feedback type="invalid">
-                  {errors?.email}
-                </Form.Control.Feedback>
-              </Form.Group>
+                <Form.Group className="mb-4" controlId="Email">
+                  <Form.Label>
+                    <h6 style={{ marginBottom: '0px' }}>Email</h6>
```

Comment:
change to the className
```suggestion
                    <h6 className="mb-0">Email</h6>
```